app/src/main/java/nerd/tuxmobil/fahrplan/congress/schedule/TrackBackgrounds.kt

```diff
@@ -0,0 +1,96 @@
+package nerd.tuxmobil.fahrplan.congress.schedule
+
+import android.content.Context
+import android.text.TextUtils
+import org.xmlpull.v1.XmlPullParser
+import java.util.HashMap
+import java.util.LinkedHashMap
+import nerd.tuxmobil.fahrplan.congress.R
+
+object TrackBackgrounds {
+    fun getHashMapResource(c: Context, hashMapResId: Int): Map<String?, String?>? {
+        var map: MutableMap<String?, String?>? = null
+        val parser = c.resources.getXml(hashMapResId)
+        var key: String? = null
+        var value: String? = null
+        try {
+            var eventType = parser.eventType
+            while (eventType != XmlPullParser.END_DOCUMENT) {
+                when (eventType) {
+//                  XmlPullParser can also be START_DOCUMENT
```

Comment:
Please move the comment `//` start to the **indentation** of the remaining code.

Reply:
This somehow ended up at the end of the while-block. I moved it back up where it belongs. :heavy_check_mark: